.github/workflows/on-push.yml

```diff
@@ -33,9 +33,9 @@ jobs:
     - name: Run Pylint
       run: |
         ./ci docker run qctrl/python-build:3.7 poetry run pylint_runner || true
-    - name: Run Pylama
+    - name: Run Black
       run: |
-        ./ci docker run qctrl/python-build:3.7 poetry run pylama || true
+        ./ci docker run qctrl/python-build:3.7 poetry run black -- --check .
```

Comment:
nit: I'd suggest putting the Black check before pylint, because Black is a lot faster, so if there are basic issues then by doing Black first we'll be able to give a result faster